client/components/data/query-email-owner-username/index.ts

```diff
@@ -0,0 +1,52 @@
+import { isTitanMail, isGoogleWorkspace } from '@automattic/calypso-products';
+import { useSelector } from 'react-redux';
+import { useQuerySitePurchases } from 'calypso/components/data/query-site-purchases';
+import useUsersQuery from 'calypso/data/users/use-users-query';
+import { getSitePurchases } from 'calypso/state/purchases/selectors';
+import type { SiteDetails } from '@automattic/data-stores';
+import type { InfiniteData } from 'react-query';
+
+type User = {
+	ID: number;
+	linked_user_ID: number;
```

Comment:
What is a _linked user_? Could we find a more explicit name which better conveys what that is?

Reply:
I'm using this type in order to TypeScript don't complain when calling the reactQuery method which is in JS (not typed). It's not my choice, its the value we are receiving thought the users endpoint API.

If its a Jetpack site, the ID is overridden and the linked_user_ID is the real WordPress user. This gave us a big headache.

Reply:
Could we then document it in some ways, either here or when we call `teams.users?.find`?